src/components/LoginPage/Login.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import "./login.css"
+import {Route, Routes, Link} from "react-router-dom";
+import Signup from '../SignUp/Signup';
+
+
+function Login() {
+  return (
+    <div>
+
+         <Routes>
+            <Route path='/Login' element={<Login /> } />
+            <Route path='/Signup' element={<Signup /> } />
+          </Routes>
+      <main>
+      <div class="page-container">
+        <h1 class="page-heading">Login Page</h1>
+        <form>
+          <label class="input-label">
+            Email Address <span class="star"> *</span></label
+          >
+          <br />
+          <input
+            class="input-box"
+            type="email"
+            placeholder="johndoe@example.com"
+          />
+          <br />
+          <br />
+
+          <label class="input-label">
+            Password <span class="star"> * </span></label
```

Comment:
Use htmlFor attribute in the label and id in the label. It will increase the touch targets.